velox/functions/prestosql/ArrayIntersectExcept.cpp

```diff
@@ -269,29 +267,145 @@ class ArrayIntersectExceptFunction : public exec::VectorFunction {
 
   // If there's a `constantSet`, whether it refers to left or right-hand side.
   const bool isLeftConstant_{false};
-};
+}; // class ArrayIntersectExcept
+
+template <typename T>
+class ArraysOverlapFunction : public exec::VectorFunction {
+ public:
+  ArraysOverlapFunction() = default;
+
+  ArraysOverlapFunction(SetWithNull<T> constantSet, bool isLeftConstant)
+      : constantSet_(std::move(constantSet)), isLeftConstant_(isLeftConstant) {}
+
+  void apply(
+      const SelectivityVector& rows,
+      std::vector<VectorPtr>& args,
+      const TypePtr& /* outputType */,
+      exec::EvalCtx* context,
+      VectorPtr* result) const override {
+    // If one of them is a ConstantVector, iterate through the other
+    // ArrayVector->elementsFlatVector and do constantSet_.find().
```

Comment:
Looks like there are some typos: ArrayVector->elementsFlatVector. Would you double check and fix as needed?